### ISSUE TITLE
Color Contrast issue is reporting as incomplete and not a violation:

### ISSUE THREAD

**Issue by scottwalter-nice** [dequelabs/axe-core#3513]
Give the following markup

<img width="959" alt="image" src="https://user-images.githubusercontent.com/59891581/174633365-d6443cb8-ca22-4cd7-8310-647aebb2c077.png">

When I run:   const results =  await window.axe.run('#root');

I get the color contrast as incomplete and not violations:

<img width="1189" alt="image" src="https://user-images.githubusercontent.com/59891581/174633654-8e3b3681-fe5e-42d0-b029-1321e843a9dd.png">

The Axe Chrome extension properly reports the color issue.

**Comment by straker** [comment]
Thanks for the issue. Is the page in question public that I could run tests on it? Or could you make a codepen or something similar that reproduces the issue?

**Comment by scottwalter-nice** [comment]
You can go here:  https://na1.dev.nice-incontact.com/nice-solaris-ngx/iframe.html?args=&id=components-button--primary&viewMode=story.  Although you will need to adjust the text color as right now that page passes color contrast.

**Comment by lvxwrkr777** [comment]
Can confirm I've seen this recently, although I can't really reveal the page that provoked it, because it's proprietary.

**Comment by straker** [comment]
I tried to change the text colors on the buttons and I get violations and 0 incomplete. I'm currently running axe-core@4.4.2. Perchance is this similar to (https://github.com/dequelabs/axe-core/issues/3464? I don't see another element inside the button elements though so maybe not.

<img width="383" alt="3 buttons of various sizes. The first and last buttons have failing color contrast, while the middle button has passing color contrast" src="https://user-images.githubusercontent.com/2433219/180242280-4fde6a53-d210-492f-840a-9f919bc56b8c.png">

**Comment by scottfwalter** [comment]
I just ran again and set the text color to be the same as the background color  (#007AB8) and it comes back in the incomplete section.  I am using axe-core 4.4.2:


<img width="1389" alt="image" src="https://user-images.githubusercontent.com/1752854/180828931-9cf08efe-f865-4b62-b0c1-c1030039a941.png">

**Comment by straker** [comment]
If you set the foreground and background color to be the same color, we mark that as incomplete. If you open up the `nodes` array and look at the message of the node in question, it should say "Element has a 1:1 contrast ratio with the background"

**Comment by straker** [comment]
@scottfwalter was the issue setting the background and foreground colors to the same color? Is this issue resolved and can be closed?

**Comment by scottfwalter** [comment]
> @scottfwalter was the issue setting the background and foreground colors to the same color? Is this issue resolved and can be closed?

Yes, it was.  Feel free to close.

**Comment by Saravan004** [comment]
@straker Why does the Color contrast is reporting as Incomplete , when it has 1:1 contrast ratio? Should it be a violation?

**Comment by WilcoFiers** [comment]
@Saravan004 When text has a 1:1 contrast that basically just means its invisible. There are lots of ways / reasons text can be made invisible. It really depends on what's being hidden and why whether there's an accessibility problem. Sometimes text is hidden to add accessibility information. Because can't be sure, axe-core doesn't fail text when it has a 1:1 contrast with the background.
